src/current/v23.1/connection-pooling.md

```diff
@@ -66,7 +114,7 @@ Validating connections is typically handled automatically by the connection pool
 
 <section class="filter-content" markdown="1" data-scope="java">
 
-In this example, a Java application similar to the [basic JDBC example](build-a-java-app-with-cockroachdb.html) uses the [PostgreSQL JDBC driver](https://jdbc.postgresql.org/) and [HikariCP](https://github.com/brettwooldridge/HikariCP) as the connection pool layer to connect to a CockroachDB cluster. The database is being run on 10 cores across the cluster.
+In this example, a Java application similar to the [basic JDBC example](build-a-java-app-with-cockroachdb.html) uses the [PostgreSQL JDBC driver](https://jdbc.postgresql.org/) and [HikariCP](https://github.com/brettwooldridge/HikariCP) as the connection pool layer to connect to a {{ site.data.products.dedicated }} CockroachDB cluster. The database is being run on 10 cores across the cluster.
```

Comment:
'CockroachDB Dedicated CockroachDB cluster' ... dang. I think I saw some docs style team discussion about this fly by but didn't get it until now.  Anyway hope we revisit this one someday